### PR TITLE
qemu 7.0.0

### PR DESCRIPTION
**Pull request by tylinux**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/qemu.rb [hunk 1]
```diff
@@ -1,10 +1,9 @@
 class Qemu < Formula
   desc "Emulator for x86 and PowerPC"
   homepage "https://www.qemu.org/"
-  url "https://download.qemu.org/qemu-6.2.0.tar.xz"
-  sha256 "68e15d8e45ac56326e0b9a4afa8b49a3dfe8aba3488221d098c84698bca65b45"
+  url "https://download.qemu.org/qemu-7.0.0.tar.xz"
```
Comment (AkihiroSuda) [inline Formula/qemu.rb:4]: QEMU 7.0 is known to cause host kernel panic on M1 (Max and Pro only?) version of macOS 12.3 when the guest RAM exceeds 4 GiB. The kernel panic is reported to be fixed in macOS 12.4 Beta: https://github.com/lima-vm/lima/issues/795

Maybe better to hold this PR until the release of macOS 12.4?
Or at least it should print a caveat message?

Comment (tylinux) [reply Formula/qemu.rb:4]: Thank you for the information, maybe we should upgrade to 7.0.0 after macOS 12.4 released.

**Comment by tylinux** [comment]
Maybe we should upgrade to 7.0.0 after macOS 12.4 released.

File: Formula/qemu.rb (hunk 1 above)
Comment (tbutter) [reply Formula/qemu.rb:4]: according to https://gitlab.com/qemu-project/qemu/-/issues/903#note_876417804 this also happens with the current version in homebrew - so it should not block 7.0.0

Comment (AkihiroSuda) [reply Formula/qemu.rb:4]: According to https://github.com/lima-vm/lima/pull/796#issuecomment-1104026408 https://github.com/lima-vm/lima/pull/796#issuecomment-1104041403 , 7.0.0 crashes but 6.2.1 does not,

I guess https://gitlab.com/qemu-project/qemu/-/issues/903#note_876417804 is hitting a different code path.

Comment (tbutter) [reply Formula/qemu.rb:4]: I think its "6.2.0 with homebrew patches" that crashes. homebrew contains several backported 7.0.0 patches

Comment (AkihiroSuda) [reply Formula/qemu.rb:4]: https://github.com/lima-vm/lima/pull/796#issuecomment-1104026408 says “ brew 6.2.0_1” does not crash.

Probably https://gitlab.com/qemu-project/qemu/-/issues/903#note_876417804 is about slightly different command line options?

Comment (tbutter) [reply Formula/qemu.rb:4]: sorry, 6.2.0_1 is brew. seems you are right.

**Comment by bb010g** [comment]
To help avoid duplicates, could this PR be reopened and marked as a draft until macOS 12.4 releases?

**Comment by SMillerDev** [comment]
Brew tooling will already avoid duplication by linking people here

**Comment by chenrui333** [comment]
#101585

**Comment by AkihiroSuda** [comment]
macOS 12.4 is now out

**Comment by SMillerDev** [comment]
```
2022-05-17T11:33:08.1204165Z ../linux-user/fd-trans.c: In function 'host_to_target_data_vlan_list_nlattr':
2022-05-17T11:33:08.1204660Z ../linux-user/fd-trans.c:853:10: error: 'IFLA_VF_VLAN_INFO' undeclared (first use in this function); did you mean 'IFLA_VF_VLAN'?
2022-05-17T11:33:08.1204967Z   853 |     case IFLA_VF_VLAN_INFO:
2022-05-17T11:33:08.1205167Z       |          ^~~~~~~~~~~~~~~~~
2022-05-17T11:33:08.1205352Z       |          IFLA_VF_VLAN
```

**Comment by tylinux** [comment]
> ```
> 2022-05-17T11:33:08.1204165Z ../linux-user/fd-trans.c: In function 'host_to_target_data_vlan_list_nlattr':
> 2022-05-17T11:33:08.1204660Z ../linux-user/fd-trans.c:853:10: error: 'IFLA_VF_VLAN_INFO' undeclared (first use in this function); did you mean 'IFLA_VF_VLAN'?
> 2022-05-17T11:33:08.1204967Z   853 |     case IFLA_VF_VLAN_INFO:
> 2022-05-17T11:33:08.1205167Z       |          ^~~~~~~~~~~~~~~~~
> 2022-05-17T11:33:08.1205352Z       |          IFLA_VF_VLAN
> ```

Looks like `ghcr.io/homebrew/ubuntu16.04:master` use `linux-libc-dev:4.4.0`, but  `IFLA_VF_VLAN_INFO` was added after linux 4.9.0-rc1, commit is here:[net: Update API for VF vlan protocol 802.1ad support
](https://github.com/torvalds/linux/commit/79aab093a0b5370d7fc4e99df75996f4744dc03f).

File: Formula/qemu.rb [hunk 2]
```diff
@@ -140,6 +65,7 @@ def install
       --enable-virtfs
       --extra-cflags=-DNCURSES_WIDECHAR=1
       --disable-sdl
+      --disable-linux-user
```
Comment (AkihiroSuda) [inline Formula/qemu.rb:68]: Could you add a comment line to explain that this should be enabled again when the CI gets updated to a recent version of Ubuntu?

**Review by SMillerDev — APPROVED** [review]

**Comment by SMillerDev** [comment]
Thanks @tylinux  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**